backend/src/main/java/ca/bc/gov/hlth/mohums/controller/ExceptionController.java

```diff
@@ -0,0 +1,28 @@
+package ca.bc.gov.hlth.mohums.controller;
+
+import ca.bc.gov.hlth.mohums.exceptions.HttpUnauthorizedException;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ControllerAdvice
+public class ExceptionController {
+
+    @ExceptionHandler({HttpUnauthorizedException.class})
+    @ResponseBody
+    @ResponseStatus(value = HttpStatus.UNAUTHORIZED)
+    ResponseEntity<Object> unauthorizedAccess(Exception e) {
+
+        HttpHeaders responseHeaders = new HttpHeaders();
+        responseHeaders.set("WWW-Authenticate",
+                "Bearer error=\"insufficient_scope\", " +
```

Comment:
Set a header that matches the response the spring security library uses when role validation fails at the spring level